composer/yamls/models/bert-base.yaml

```diff
@@ -70,13 +70,17 @@ precision: amp                     # Use mixed-precision training
 grad_clip_norm: -1.0               # Turn off gradient clipping
 grad_accum: 'auto'                 # Use automatic gradient accumulation to avoid OOMs
 
-save_folder: bert_checkpoints      # The directory to save checkpoints to
+save_folder: checkpoints          # The directory to save checkpoints to
 save_interval: 3500ba              # Save checkpoints every 3500 batches
+save_artifact_name: '{run_name}/checkpoints/ep{epoch}-ba{batch}-rank{rank}-wct{total_wct}' # uncomment if you want to checkpoint to cloud 
 
 loggers:
   progress_bar: {}                 # Add a TQDM progress bar
-  # Optional, some nice default parameters for object store checkpointing. Uncomment this if you want to checkpoint to cloud.
-  # object_store:
-  #  object_store_hparams:
-  #    s3:
-  #      bucket: mosaicml-internal-checkpoints-bert
+  # Optional, some nice default parameters for object store checkpointing and WandB logging. Uncomment these if you want to checkpoint to cloud and track training info.
+  object_store:
+    object_store_hparams:
+      s3:
+        bucket: mosaicml-internal-checkpoints-bert
```

Comment:
Since this YAML is public facing, seems like maybe we would want to use a different bucket. Also this feels like the kind of thing that you wouldn't want to be part of a default BERT config. 

Is it possible to simply have the entrypoint activate this kind of storage, rather than make it active by default?

Reply:
Sure, we can remove it. In the documentation, I can make a copy of this yaml to demonstrate how to use the entrypoint with S3/WandB.

Reply:
OK great. In general, I think we want to avoid directly referencing this `composer/yamls/model/bert-base.yaml`, since it may get confusing when people are using this entrypoint with their own pre-training YAML.